avm-transpiler/src/transpile.rs

```diff
@@ -460,7 +454,6 @@ fn handle_emit_unencrypted_log(
         // The message array from Brillig is indirect.
         indirect: Some(FIRST_OPERAND_INDIRECT),
```

Comment:
Ah I think you need to change this to `ZEROTH_OPERAND_INDIRECT` because we removed the zeroth-operand.

If this is wrong it should fail in the avm_simulator.test.ts.... ideally